onnxruntime/core/providers/acl/acl_common.cc

```diff
@@ -0,0 +1,63 @@
+// Copyright(C) 2018 Intel Corporation
+// Copyright (c) 2019, NXP Semiconductor, Inc. All rights reserved.
+// Licensed under the MIT License
+
+#ifdef _WIN32
+#pragma warning(disable : 4244)
+#endif
+
+#include "core/providers/acl/acl_common.h"
+
+#include "arm_compute/runtime/PoolManager.h"
+#include "arm_compute/runtime/BlobLifetimeManager.h"
+
+#undef ACL_1902
```

Comment:
https://github.com/microsoft/onnxruntime/blob/67755adfd840f31209b46ffd6d0263ea129d88ef/cmake/CMakeLists.txt#L549-L551

Reply:
Thanks, I will expose this configuration to cmake in the next patchset.